stdlib/public/TensorFlow/TensorGroup.swift

```diff
@@ -130,3 +147,18 @@ extension TensorElementLiteral : TensorGroup {
     self.init(handle: TensorHandle(_owning: tensorHandles!.pointee))
   }
 }
+
+extension Array: InputTensorGroup where Element: InputTensorGroup {
```

Comment:
The convention in the stdlib is to put spaces on both sides of the `:` for protocol conformances.